subprojects/soak/src/integTest/groovy/org/gradle/launcher/daemon/DaemonPerformanceMonitoringSoakTest.groovy

```diff
@@ -123,7 +123,7 @@ class DaemonPerformanceMonitoringSoakTest extends DaemonMultiJdkIntegrationTest
     }
 
     def "when build leaks permgen space daemon is expired"() {
-        assumeTrue(version.vendor != JdkVendor.IBM && version.version == "1.7")
+        assumeTrue(version.vendor != JdkVendor.IBM)
```

Comment:
So this test did not work for me when I tried it against a Java8 jdk (note that there's a fix for the initial heap problem on release).  It might not be aggressive enough or perhaps what we're doing is not appropriate to simulate a leak in the metaspace pool - not sure.  This was tuned to cause a leak within 20 builds for Java7, but the behavior of Java8 metaspace might be different.  Let's chat about it.

Reply:
So it looks like it's just that the default max metaspace that we use is large enough that this test doesn't exhaust it fast enough.  Note that with Java7, perm gen was based off the max heap size, so the smaller heap set for this test would trigger it.  When I set the max metaspace to the same as the heap size (200m), it seems to reliably expire the daemon in about 15 builds.